tests/test

```diff
@@ -13,5 +13,10 @@
 # To run only one test:
 #     $ cd tests && ./test -run=TestEnv
 
-sudo GOPATH="${PWD}/../gopath" RKT_ENABLE_DESTRUCTIVE_TESTS="$RKT_ENABLE_DESTRUCTIVE_TESTS" go test -v $*
+# Sudo does not preserve $PATH. Use go from the user's $PATH instead of
+# root's $PATH. It is needed when a home-built go does not have the same
+# version as the system-wide installed go.
+GO=`which go`
```

Comment:
I guess you refer to `sudo su`, `sudo su -` doesn't preserve the environment. Anyway, $PATH is a special case and does not get preserved with `sudo su`.